gator/files.py

```diff
@@ -23,10 +23,45 @@ def create_path(*args, file, home):
     return final_path
 
 
-def check_file_in_directory(*args, file, home):
-    """Returns true if the specified file is in the directory"""
+def case_insensitive_check_file_in_directory(*args, file, home):
+    """Returns true if the case-insensitive specified file is in the directory"""
     # create the path so that it has this structure:
     # <home> + <any paths in *args, without their anchor> + <file>
     file_for_checking_path = create_path(*args, file=file, home=home)
```

Comment:
On Linux this will still be a case-sensitive check, right? Does more logic need to be added here, or do paths refactor things to be case-insensitive by default? Alternatively, a better name might be needed that reflects this uses the OS's default, and is not always case-insensitive.

Reply:
Hi @Michionlion I think that this function works correctly, but the name of the function is not correct. That is, this is doing whatever the operating system would run as the check. So, it is *potentially* case insensitive, which is what was in my mind but not in the code!

Reply:
I will fix that then; the CI also needs to be rerun so that will take care of both of those issues!